pkg/routeagent_driver/cni/cni_iface.go

```diff
@@ -79,7 +79,7 @@ func ConfigureRpFilter(iface string) error {
 	// #nosec G306
 	err := ioutil.WriteFile("/proc/sys/net/ipv4/conf/"+iface+"/rp_filter", []byte("2"), 0644)
 	if err != nil {
-		return fmt.Errorf("unable to update rp_filter for cni_interface %q, err: %s", iface, err)
+		return fmt.Errorf("unable to update rp_filter for cni interface %q, err: %s", iface, err)
```

Comment:
CNI?

Reply:
This applies to all `cni`s throughtout the codebase and its out of the scope of this patch.

Reply:
This applies to all `CNI Interfaces` throughtout the code and is out of the scope of this PR.

Reply:
Kinda, I mean we are adding this comment and introducing a new "cni" that should be "CNI". As @skitt suggested, probably best to raise an issue to track adding additional config for custom acronyms we want to watch for.

Reply:
It's referred to in several different ways in messages in this file, ie _CNI Interface_, _cni_interface_, _cniInterface_, _cniIface_. We should be consistent - IMO _CNI_ as @dfarrell07 suggest. However I do agree it's a bit out of scope for this PR - this PR is already quite large and refactors quite a bit of files so it would be nice to get this merged ASAP. I have work to do in several of the files but have been holding off to avoid significant conflict resolution.

Reply:
@Jaanki Can  you address this in another PR?